server/pulp/server/managers/schedule/utils.py

```diff
@@ -98,7 +98,7 @@ def delete(schedule_id):
     try:
         spec = {'_id': ObjectId(schedule_id)}
     except InvalidId:
-        raise exceptions.InvalidValue(['schedule_id'])
+        raise exceptions.MissingResource(schedule_id=schedule_id)
```

Comment:
Add a comment here on why this is being converted to a MissingResource rexception